assets/js/translator.js

```diff
@@ -4,10 +4,39 @@ var curSrcLang, curDstLang;
 var recentSrcLangs = [], recentDstLangs = [];
 var droppedFile;
 var textTranslateRequest;
+var recaptchaScriptSrc = "https://www.google.com/recaptcha/api.js?onload=recaptchaRenderCallback&render=explicit&hl=";
+
+var recaptchaRenderCallback = function () {
+    grecaptcha.render('suggestRecaptcha', {
+      'sitekey' : config.SUGGESTIONS.recaptcha_site_key
+    });
+}
 
 if(modeEnabled('translation')) {
     $(document).ready(function () {
         synchronizeTextareaHeights();
+        var localLang = iso639Codes[$('.localeSelect').val()];
+        var newSrc = recaptchaScriptSrc + 'en';
+        var backoff = false;
+
+        for(var i = 0; i < localizeRecaptcha.length; i++) {
+            if (localLang == localizeRecaptcha[i]) {
+                newSrc = recaptchaScriptSrc + localLang;
+                backoff = false;
+                break
+            }
+            else {
+                backoff = true;
+            } 
+        }
+
+        if (backoff) {
+            newSrc = recaptchaScriptSrc + localizeRecaptchaAlternative[localLang]; 
+        }
```

Comment:
We're duplicating the exact same logic here :( Can we create a: `function pickRecaptchaLanguage(locale) { ... }` in `localization.js`.